### PR TITLE
OPSRC 679 Add OS Header to ECS

### PR DESCRIPTION
**Pull request by gabriela-lubkowska**
- Adds BitBag OS Header to ECS
- Excludes Migrations directory and Kernel.php file from using header rule

### CODE REVIEW

File: ecs.php [hunk 1]
```diff
@@ -395,4 +396,14 @@
     $services->set(FinalClassInEntitiesFixer::class);
 
     $services->set(AboveTwoArgumentsMultilineFixer::class);
+
+    $services
+        ->set(HeaderCommentFixer::class)
+        ->call('configure', [[
+            "header" => "This file has been created by developers from BitBag. \nFeel free to contact us once you face any issues or want to start\nYou can find more information about us on https://bitbag.io and write us\nan email on hello@bitbag.io.",
```
Comment (senghe) [inline ecs.php]: I would put the content into constant.
Additionally, I would make environment variable to disable this check due to no need for OS header in commercial and private projects.

**Review by senghe — CHANGES_REQUESTED** [review]

File: ecs.php (hunk 1 above)
Comment (gabriela-lubkowska) [reply ecs.php]: We decided to not put header content in the constant. Changes regarding adding environment variable are done in next commit.

File: README.md [hunk 2]
```diff
@@ -57,6 +57,10 @@ use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigura
 use Symplify\EasyCodingStandard\ValueObject\Option;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
+
+    // Change this if you want to automatically add BitBag OpenSource Header
+    putenv('ALLOW_BITBAG_OS_HEADER=0');
```
Comment (senghe) [inline README.md:62]:
```suggestion
    putenv('ALLOW_BITBAG_OS_HEADER=1');
```

**Review by senghe — APPROVED** [review]
All good except the one, last comment. But I'm giving you approve in advance. Please change the line I suggested and then you can merge it :)

File: README.md [hunk 3]
```diff
@@ -57,6 +57,10 @@ use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigura
 use Symplify\EasyCodingStandard\ValueObject\Option;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
+
+    // Change this if you don't want to automatically add BitBag OpenSource Header
+    putenv('ALLOW_BITBAG_OS_HEADER=1');
```
Comment (senghe) [inline README.md]:
```suggestion
    putenv('ALLOW_BITBAG_OS_HEADER=0');
```

File: README.md [hunk 4]
```diff
@@ -57,6 +57,10 @@ use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigura
 use Symplify\EasyCodingStandard\ValueObject\Option;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
+
+    // Change this if you don't want to automatically add BitBag OpenSource Header
```
Comment (senghe) [inline README.md]:
```suggestion
    // Change this if you want to automatically add BitBag OpenSource Header
```

**Pull request merged**